Cython/Compiler/Pipeline.py

```diff
@@ -173,8 +173,10 @@ def create_pipeline(context, mode, exclude_classes=()):
 
     if mode == 'py':
         _align_function_definitions = AlignFunctionDefinitions(context)
+        _auto_cpdef_function_definitions = AutoCpdefFunctionDefinitions(context)
     else:
         _align_function_definitions = None
+        _auto_cpdef_function_definitions = None
```

Comment:
Is there a good reason (apart from history) why `auto_cpdef` is ignored in .pyx files? And if there is, then should we warn when someone tries to use it?

It looks to be undocumented so hopefully not too many people are using it of course

Reply:
Good point. There is no particular reason for that. I have fixed that.